mmpose/datasets/pipelines/mesh_transform.py

```diff
@@ -0,0 +1,370 @@
+import cv2
+import mmcv
+import numpy as np
+from torchvision.transforms import functional as F
+
+from mmpose.core.post_processing import (affine_transform, fliplr_joints,
+                                         get_affine_transform)
+from mmpose.datasets.registry import PIPELINES
+
+
+def flip_smpl_pose(pose):
+    """Flip SMPL pose parameters horizontally.
```

Comment:
what does `SMPL` mean

Reply:
SMPL is a 3D human model that takes pose parameters and shape parameters as input and output a 3D human mesh.

Reply:
note: https://smpl.is.tue.mpg.de/